indexer/services/comlink/src/controllers/api/v4/compliance-controller.ts

```diff
@@ -0,0 +1,82 @@
+import { stats } from '@dydxprotocol-indexer/base';
+import express from 'express';
+import { checkSchema, matchedData } from 'express-validator';
+import {
+  Controller, Get, Query, Route,
+} from 'tsoa';
+
+import { getReqRateLimiter } from '../../../caches/rate-limiters';
+import config from '../../../config';
+import { handleControllerError } from '../../../lib/helpers';
+import { rateLimiterMiddleware } from '../../../lib/rate-limit';
+import ExportResponseCodeStats from '../../../request-helpers/export-response-code-stats';
+import { ComplianceRequest, ComplianceResponse } from '../../../types';
+
+const router: express.Router = express.Router();
+const controllerName: string = 'compliance-controller';
+
+@Route('screen')
+class ComplianceController extends Controller {
+  @Get('/')
+  screen(
+    @Query() address: string,
+  ): ComplianceResponse {
+    // TODO(IND-372): Add logic to either use cached data or query provider
+    // Dummy logic for front-end testing
+    if (
+      address.charCodeAt(address.length - 1) > 'a'.charCodeAt(0) &&
+      address.charCodeAt(address.length - 1) < 'm'.charCodeAt(0)
+    ) {
```

Comment:
nit: logic is a little confusing, which you elaborated on in git commit message, could you include that in a comment here?